### PR TITLE
feat(quantic): migrate all tests to community

### PR DESCRIPTION
**Pull request by lbergeron**
https://coveord.atlassian.net/browse/SFINT-4043

This pull request builds on top of [feat(quantic): add component specific end-to-end tests using Salesforce example community](https://github.com/coveo/ui-kit/pull/1027).

It contains the following changes:

- adds a page to the community containing the example search component
- updates `example-search.cypress.ts` to use the newly added community page
- reworks the aliases/selectors to make tests simpler
- moves test files at the same level
- introduces a generic configurator component with Cypress-friendly utilities. We won't have to code specific configurator components like in the parent PR.
- updates quantic readme (a step was missing)

### CODE REVIEW

File: packages/quantic/README.md [hunk 1]
```diff
@@ -60,23 +60,29 @@ Example components are available as a Salesforce community (Digital Experience)
     sfdx force:community:create -u LWC -n "Quantic Examples" -p examples -t "Build Your Own"
     ```
 
-2. To deploy the community in the scratch org, run:
+2. To deploy the Quantic and examples components, run:
```
Comment (jpmarceau) [inline packages/quantic/README.md:63]:
```suggestion
2. To deploy Quantic and the example components, run:
```

Comment (jpmarceau) [reply packages/quantic/README.md:63]: Unless you meant: `To deploy the Quantic example components, run:`

Comment (lbergeron) [reply packages/quantic/README.md:63]: Your first suggestion is perfect! Thanks

**Review by jpmarceau — APPROVED** [review]

**Review by jeremierobert-coveo — APPROVED** [review]

File: packages/quantic/cypress/integration/example-search.cypress.ts [hunk 2]
```diff
@@ -1,42 +1,63 @@
...
 } from '../page-objects/example-search';
 
 describe('example-search', () => {
-  const exampleSearchUrl = 'http://localhost:3334/preview/c/exampleSearch';
+  const pageUrl = `${Cypress.env('examplesUrl')}/s/full-search-example`;
+
+  Cypress.on('uncaught:exception', (err) => {
+    if (
+      err.message.includes(
+        'navigator.sendBeacon is not supported in this browser'
```
Comment (tedre191) [inline packages/quantic/cypress/integration/example-search.cypress.ts:16]: This is to suppress the error? Why is this needed? Might be worth adding a comment to explain.

**Review by tedre191 — APPROVED** [review]
Looks nice! :gg:

File: packages/quantic/force-app/examples/main/lwc/exampleQuanticFacet/exampleQuanticFacet.js [hunk 3]
```diff
@@ -2,9 +2,25 @@ import {api, LightningElement, track} from 'lwc';
 
 export default class ExampleQuanticFacet extends LightningElement {
     @api engineId = 'quantic-facet-engine';
-
-    isConfigured = false;
     @track config = {};
+    isConfigured = false;
+
+    pageTitle = 'Quantic Facet';
+    pageDescription = 'The Quantic Facet allows to refine search results by selecting one or more field values.'
```
Comment (nathanlb) [inline packages/quantic/force-app/examples/main/lwc/exampleQuanticFacet/exampleQuanticFacet.js:9]:
```suggestion
    pageDescription = 'The Quantic Facet allows you to refine search results by selecting one or more field values.'
```

**Review by nathanlb — APPROVED** [review]

File: packages/quantic/force-app/examples/main/lwc/exampleQuanticFacet/exampleQuanticFacet.js (hunk 3 above)
Comment (lbergeron) [reply packages/quantic/force-app/examples/main/lwc/exampleQuanticFacet/exampleQuanticFacet.js:9]: I think `...allows users to refine...` reads better. Thanks!

File: packages/quantic/cypress/integration/example-search.cypress.ts (hunk 2 above)
Comment (lbergeron) [reply packages/quantic/cypress/integration/example-search.cypress.ts]: @tedre191 The error comes from [Coveo.analytics.js](https://github.com/coveo/coveo.analytics.js/blob/master/src/client/analyticsBeaconClient.ts#L21). Basically, it happens because the browser when the browser switches between tests, the UA event buffer might not have been emptied. So, when Coveo.analytics.js tries to flush the events, `navigator.sendBeacon` does not exist anymore.

Now, I think we can safely ignore it because, either the test does not assert UA requests, or it will wait for UA requests to be made (preventing this error at the same time).
